src/models/FGInertial.cpp

```diff
@@ -217,11 +223,31 @@ void FGInertial::SetAltitudeAGL(FGLocation& location, double altitudeAGL)
 
 //%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+void FGInertial::SetGravityType(int gt)
+{
+  // Messages to warn the user about possible inconsistencies.
+  switch (gt)
+  {
+  case eGravType::gtStandard: 
+    if (a != b)
+      cout << "Set standard gravity for a non-spherical planet" << endl;
+    break;
+  case eGravType::gtWGS84:
+    if (J2 == 0.0)
+      cout << "Set WGS84 with a null gravitational constant J2." << endl;
```

Comment:
Suggest the following:

"Warning: WGS84 gravity model has been set without specifying the J2 gravitational constant."

Reply:
Done.